e2e/test/scenarios/filters/filter.cy.spec.js

```diff
@@ -560,26 +560,26 @@ describe("scenarios > question > filter", () => {
     cy.findByText("wilma-muller");
   });
 
-  it("should not reject a number literal", () => {
+  it("should reject a number literal", () => {
```

Comment:
Reverts previous invalid changes. This is how it was in master, minus eslint fixes